src/components/StoriesPage/index.js

```diff
@@ -20,57 +21,92 @@ const useStyles = makeStyles(({ typography, breakpoints }) => ({
 
 function StoriesPage({
   activeCategory,
-  items,
+  items: itemsProp,
   hero,
   featuredStories,
   ...props
 }) {
   const classes = useStyles(props);
   const contentRef = useRef();
+  const [category, setCategory] = useState(activeCategory);
+  const [items, setItems] = useState(itemsProp);
   const [page, setPage] = useState(1);
+  const [shouldFetch, setShouldFetch] = useState(false);
+
+  const { data, error } = useSWR(
+    shouldFetch ? "/api/wp/archive" : null,
```

Comment:
`category` and `offset` needs to be part of arguments sent to fetcher (alongside `url`). Otherwise, [`SWR`](https://swr.vercel.app/docs/arguments#multiple-arguments) sees `"/api/wp/archive"` as a constant and caches the results even when `page` & `category` changes.